docs/Using Fleet/Automations.md

```diff
@@ -4,6 +4,12 @@ You can configure Fleet to trigger automations that reserve time in your end use
 
 To learn how to use Fleet's maintenance windows, head to this [article](https://fleetdm.com/announcements/fleet-in-your-calendar-introducing-maintenance-windows). 
 
+## Activity automations
+
+Activity automations are triggered when an activity happens in Fleet (queries, scripts, logins, etc). See a list of all activities [here](../Using%20Fleet/Audit-logs.md).
```

Comment:
@eashaw does this kind of link work when the docs are built on the website?

Reply:
@rachaelshaw No, it does not. When built for the website, the link would go to `fleetdm.com/docs/using%20fleet/audit-logs`.
I'll create an issue to update the build script to support relative links to the `docs/using fleet/` folder.

```suggestion
Activity automations are triggered when an activity happens in Fleet (queries, scripts, logins, etc). See a list of all activities [here](https://fleetdm.com/docs/using-fleet/audit-logs).
```

Reply:
^cc @noahtalerman

Reply:
Ah, thanks for catching that Rachael. Good to know.